dagshub/common/cli.py

```diff
@@ -52,6 +55,106 @@ def mount(ctx, verbose, **kwargs):
     mount(**kwargs)
 
 
+@cli.command()
+@click.option("--repo_name", help="The repository name to set up")
+@click.option("--repo_owner", help="The organization under which the repository needs to be set up")
+@click.option("--url", help="DagsHub remote url; either provide --url or repo_name and repo_owner")
+@click.option("--host", default=config.DEFAULT_HOST, help="DagsHub instance to which you want to login")
+@click.option("--mlflow", default=True, is_flag=True, help="Set up MLFlow environment with DagsHub remote")
+@click.option("--dvc", default=False, is_flag=True, help="Set up DVC with DagsHub remote")
+@click.pass_context
+def init(ctx, repo_name, repo_owner, url, host, mlflow, dvc):
+    _init(repo_name, repo_owner, url, host, mlflow, dvc)
+
+
+def _init(repo_name=None, repo_owner=None, url=None, root=None,
+          host=config.DEFAULT_HOST, mlflow=True, dvc=False):
+    from ..upload.wrapper import create_repo
+    import urllib
+
+    # Setup required variables
+    root = root or get_project_root(Path(os.path.abspath('.')))
+
```

Comment:
What happens if the user calls that outside of a repository while not specifying the root argument?
I see that you're throwing a ValueError with a good enough error message, but it'll be better if you print out that the user needs to do this in a repo, instead of scaring them with a stack trace